platform/mellanox/mlnx-sai.mk

```diff
@@ -19,7 +19,7 @@ export MLNX_SAI_VERSION MLNX_SAI_SOURCE_BASE_URL
 
 MLNX_SAI = mlnx-sai_1.mlnx.$(MLNX_SAI_VERSION)_$(CONFIGURED_ARCH).deb
 $(MLNX_SAI)_SRC_PATH = $(PLATFORM_PATH)/mlnx-sai
-$(MLNX_SAI)_DEPENDS += $(MLNX_SDK_DEBS) $(LIBNL_ROUTE3_DEV)
+$(MLNX_SAI)_DEPENDS += $(MLNX_SDK_DEBS) $(LIBNL3_DEV) $(LIBNL_GENL3_DEV) $(LIBNL_ROUTE3_DEV) $(LIBNL_NF3_DEV)
```

Comment:
Why now new dependency needed?

Reply:
Verified that this is redundant so it was removed